refactor: Update `README.md` and improved repository (#926)
parse-community/Parse-SDK-Flutter: packages/dart/README.md

<img src="https://user-images.githubusercontent.com/5673677/166120960-ea1f58e3-a62b-4770-b541-f64186859339.png" alt="parse-repository-header-sdk-dart" style="max-width: 100%;">

---

> [![pub package](https://img.shields.io/pub/v/parse_server_sdk.svg)](https://pub.dev/packages/parse_server_sdk) [![Pub Likes](https://img.shields.io/pub/likes/parse_server_sdk)](https://pub.dev/packages/parse_server_sdk/score) [![Pub Popularity](https://img.shields.io/pub/popularity/parse_server_sdk)](https://pub.dev/packages/parse_server_sdk/score) [![Pub Points](https://img.shields.io/pub/points/parse_server_sdk)](https://pub.dev/packages/parse_server_sdk/score)

Author: ok

[![License](https://img.shields.io/badge/license-Apache%202.0-lightgrey.svg)](https://github.com/parse-community/Parse-SDK-Flutter/blob/master/LICENSE)
[![Forum](https://img.shields.io/discourse/https/community.parseplatform.org/topics.svg)](https://community.parseplatform.org/c/parse-server)

[![Backers on Open Collective](https://opencollective.com/parse-server/backers/badge.svg)][open-collective-link]
[![Sponsors on Open Collective](https://opencollective.com/parse-server/sponsors/badge.svg)][open-collective-link]
[![Twitter Follow](https://img.shields.io/twitter/follow/ParsePlatform.svg?label=Follow%20us&style=social)](https://twitter.com/intent/follow?screen_name=ParsePlatform)
[![Chat](https://img.shields.io/badge/Chat-Join!-%23fff?style=social&logo=slack)](https://chat.parseplatform.org)

---

This library gives you access to the powerful Parse Server backend from your Dart app. For more information on Parse Platform and its features, visit [parseplatform.org](https://parseplatform.org).

---

- [Compatibility](#compatibility)

> - [Getting Started](#getting-started)

Reviewer: ```suggestion
- [Getting Started](#getting-started)
- [Documentation](#documentation)
- [Contributing](#contributing)
```

---

## Compatibility

The Parse Dart SDK is continuously tested with the most recent release of the Dart framework to ensure compatibility. To give developers time to upgrade their app to the newest Dart framework, previous Dart framework releases are supported for at least 1 year after their [release date](https://dart.dev/get-dart/archive).

| Version   | Latest Version | End of Support | Compatible                                   |
|-----------|----------------|----------------|----------------------------------------------|
| Dart 3.0  | 3.0.0          | May 2024       | ✅ Yes                                        |
| Dart 2.19 | 2.19.6         | Mar 2024       | ✅ Yes                                        |
| Dart 2.18 | 2.18.7         | Jan 2024       | ✅ Yes                                        |
| Dart 2.17 | 2.17.7         | Aug 2023       | ❌ No (Parse Dart SDK requires Dart >=2.18.0) |

## Getting Started

> To install, either add [dependency in your pubspec.yaml file](https://pub.dev/packages/parse_server_sdk/install).

Reviewer: ```suggestion
To install, add the Parse Dart SDK as a [dependency](https://pub.dev/packages/parse_server_sdk/install) in your `pubspec.yaml` file.
```

> See the SDK Usage [guide][guide].

Reviewer: ```suggestion
## Documentation

Find the full documentation in the [Parse Dart SDK guide][guide].
```

> # Contributing

Reviewer: ```suggestion
## Contributing

```
